Formula/spdx-sbom-generator.rb

```diff
@@ -0,0 +1,35 @@
+class SpdxSbomGenerator < Formula
+  desc "Support CI generation of SBOMs via golang tooling"
+  homepage "https://github.com/spdx/spdx-sbom-generator"
+  url "https://github.com/spdx/spdx-sbom-generator/archive/refs/tags/v0.0.13.tar.gz"
+  sha256 "7d088f136a53d1f608b1941362c568d78cc6279df9c1bdb3516de075cb7f10c3"
+  license any_of: ["Apache-2.0", "CC-BY-4.0"]
+
+  depends_on "go" => :build
+  depends_on "python@3.8" => :test
+
+  def install
+    if OS.mac?
+      if Hardware::CPU.intel?
+        system "make", "build-mac"
+      else
+        system "make", "build-mac-arm64"
+      end
+    else
+      system "make", "build"
+    end
+
+    bin.install "bin/spdx-sbom-generator"
+  end
+
+  test do
+    system "git", "clone", "https://github.com/JoshuaMulliken/wyzeapy.git"
+    Dir.chdir("wyzeapy") do
+      system "git", "checkout", "tags/0.2.10"
+      system "python3", "-m", "pip", "install", "pipenv"
```

Comment:
Do we need `pipenv`? Does using `venv` not work, so we don't need an extra `pip install`?

Reply:
I just changed it so that it checks itself and it is a go app so no more problems with python haha